lib/std/compress/zstandard.zig

```diff
@@ -34,7 +34,7 @@ pub fn DecompressStream(
         checksum: if (options.verify_checksum) ?u32 else void,
         current_frame_decompressed_size: usize,
 
-        pub const Error = ReaderType.Error || error{
+        pub const Error = ReaderType.Error || RingBuffer.Error || error{
```

Comment:
Why should `ReadLengthInvalid` be part of the zstd error set? If this ever occurred, it would be a bug in the zstd code, right?

Reply:
ah yes, fixed